entries/repo-website-overview.puml

```diff
@@ -0,0 +1,80 @@
+@startuml Repos-Sites
+
+  package .github as GIT {
+      diamond dia1
+  }
+
+  package data as DAT {
+      diamond dia2
+  }
+
+  package theme as THE {
+      diamond dia3
+  }
+
+  package community as COM {
+      object "Community.Seedcase-Project.org" as CSP {
```

Comment:
As Luke mentioned on the meeting, it might be an idea to use Mermaid as done in the SDCA research-ops repo [here](https://steno-aarhus.github.io/research-ops/) so we can add a link directly to the websites instead of writing the url?

Reply:
@lwjohnst86 Thoughts? Should the puml be changed to mermaid?

Reply:
Yes please! Mermaid is easier to use to incorporate clickable links.

Reply:
The re-write in Mermaid will have to wait until after my summer holiday, do you guys want to merge this in, and then I can create an issue on converting it to Mermaid, or should I move this back into draft to make it go away from my lists?